conf/mysql

```diff
@@ -0,0 +1 @@
+MYSQL_ROOT_PASSWORD=yourpassword
```

Comment:
TL:DR; I will remove it :dango: 

Some more text: 
I understand that people will not think of changing this, except when they really read and follow the tutorial, or when they launched their instances and are wondering what the root password is. 
Having this in place by default, with a acceptable default password, could benefit the startup routine as it is just a run and go situation, without any hassle of password configuration. 
If you are still thinking about adding a `configure` script, then I'd suggest including this as a step.